configure.ac

```diff
@@ -8,10 +8,10 @@
 #
 AC_PREREQ([2.63])
 
-define([ucx_ver_major], 1)  # Major version. Usually does not change.
-define([ucx_ver_minor], 14) # Minor version. Increased for each release.
-define([ucx_ver_patch], 0)  # Patch version. Increased for a bugfix release.
-define([ucx_ver_extra], "") # Extra version string. Empty for a general release.
+define([ucx_ver_major], 1)       # Major version. Usually does not change.
+define([ucx_ver_minor], 14)      # Minor version. Increased for each release.
+define([ucx_ver_patch], 0)       # Patch version. Increased for a bugfix release.
+define([ucx_ver_extra], ".pre1") # Extra version string. Empty for a general release.
```

Comment:
i would put all fixes, except this one, to master (and v1.13) branch
and in integration4, keep only the commit to set ucx_ver_extra to ".pre1"

Reply:
yeah, that's the plan, I want to test the whole patch here before backport 

> in integration4, keep only the commit to set ucx_ver_extra to ".pre1"

and I think + updated `%changelog` section in ucx.spec.in